timeutil/human.go

```diff
@@ -78,3 +78,9 @@ func humanTimeSince(then, now time.Time, cutoffDays int) string {
 		panic("you have broken the law of trichotomy! ℤ is no longer totally ordered! chaos ensues!")
 	}
 }
+
+func MockTimeNow(f func() time.Time) (restorer func()) {
```

Comment:
as this is exported now it probably merits a doc comment, look how we formulate them for this kind of Mock helper in other package.s